.circleci/config.yml

```diff
@@ -27,6 +27,15 @@ jobs:
           command: |
             if [ "${CIRCLE_BRANCH}" == "master" ]; then
               cd ~/cumulus-message-adapter
+
+              export VERSION=$(jq --raw-output .version package.json)
+              curl -H \
+                "Authorization: token $GITHUB_TOKEN" \
+                -d "{\"tag_name\": \"v$VERSION\", \"target_commitsh\": \"v$VERSION\", \"name\": \"v$VERSION\"}" \
+                -H "Content-Type: application/json" \
+                -X POST \
+                https://api.github.com/repos/nasa/cumulus-message-adapter-js/releases |grep \"url\" |grep releases |sed -e 's/.*\(https.*\)\"\,/\1/'| sed -e 's/api/uploads/'
```

Comment:
Thanks for addressing. Is this going to publish every time a change is pushed to master, regardless of whether it is a new version? I guess that would have worked for npm since it denies publish of an existing version, but how does Github react to a POST of an existing release?